bsp4j/src/main/java/ch/epfl/scala/bsp4j/BuildServer.java

```diff
@@ -46,6 +46,9 @@
     @JsonRequest("buildTarget/cleanCache")
     CompletableFuture<CleanCacheResult> buildTargetCleanCache(CleanCacheParams params);
 
+    @JsonRequest("buildTarget/executionEnvironment")
+    CompletableFuture<ExecutionEnvironmentResult> getClasspath(ExecutionEnvironmentParams params);
```

Comment:
Should we move this to `ScalaBuildServer` and call it `buildTarget/javaExecutionEnvironment`? We try to keep the language agnostic APIs out of the core BSP spec.

Reply:
To add to that, what do you think about adding more specific execution environments instead of a single request? I don't think a build target has a unique execution environment. For example, the environment to run an application or run tests can be different. I would go for `buildTarget/javaRunEnvironment`  and `buildTarget/javaTestEnvironment` instead.

Reply:
As for now, I made  `buildTarget/javaTestEnvironment` only